server/src/utils/fight/fightMethods.ts

```diff
@@ -1374,6 +1407,19 @@ const breakShield = (fighter: DetailedFighter, opponent: DetailedFighter) => {
   return getFighterStat(fighter, 'disarm') * 100 >= randomBetween(1, 300);
 };
 
+const dropShield = (fightData: DetailedFight, fighter: DetailedFighter) => {
+  // Remove brute's shield
+  fighter.shield = false;
+  fighter.skills = fighter.skills.filter((sk) => sk.name !== SkillName.shield);
+  fighter.block -= SkillModifiers[SkillName.shield][FightStat.BLOCK]?.percent ?? 0;
```

Comment:
I made a typo on the way I handle block stat changes, it should now be
```suggestion
  fighter.block -= (SkillModifiers[SkillName.shield][FightStat.BLOCK]?.percent ?? 0) / 100;
```